src/condor_dagman/job.cpp

```diff
@@ -574,6 +574,10 @@ Job::CanAddParent( Job* parent, std::string &whynot )
 		whynot = "Tried to add a parent to a Final node";
```

Comment:
Now that we have three types here, how about we turn this into a switch statement?